client/state/ui/guided-tours/selectors/index.js

```diff
@@ -149,7 +149,7 @@ const findTriggeredTour = ( state ) => {
 	} );
 };
 
-const isSectionBlacklisted = ( state ) => includes( BLACKLISTED_SECTIONS, getSectionName( state ) );
+const isSectionDisallowed = ( state ) => includes( DISALLOWED_SECTIONS, getSectionName( state ) );
```

Comment:
I stumbled upon how this expression sounds but I couldn't come up with a better name either. _Blocked_ doesn't really make sense to me in this context.

Reply:
I remember having a hard time with it as well 😞 I took another look at the context, maybe `doesSectionAllowTours` and then we just reverse the usage. I think part of the difficulty is that the function asserts a negative. Usually I like to have functions assert a positive so that there's no chance of a double negative as in the case of `! isSectionDisallowed` here.

I'll change it to `doesSectionAllowTours` and see if it makes more sense 😊

Reply:
sounds good to me 🍏

Reply:
Using the include/exclude vocabulary also looks like a good fit to me. `EXCLUDED_SECTIONS`, `isSectionExcluded`.